prisma/schema.prisma

```diff
@@ -22,7 +22,7 @@ model Transfer {
   sender       String?
   destination  String?
   amount       String?
-  timestamp    Int?
+  timestamp    Float?
```

Comment:
How is this possible?

Reply:
Becuase of timestamps on substrate. Number is bigger there and mode wasn't allowed to store the timestamp

Reply:
Pls convert to utc everything so it is all the same

Reply:
done